src/components/Header/Header.js

```diff
@@ -17,4 +18,11 @@ const Header = ({name, quote, ranking, logout}) => {
   )
 }
 
+Header.PropTypes = {
```

Comment:
@JamesRexMiller4 I'm getting an error on this line because of a typo, it should be 'propTypes' rather than 'PropTypes'. Could you please fix that? It only seems to have a problem with this line and not the others.

Reply:
Good catch. I will fix this